cypress/utils/AutoScrollableMenuSelected.js

```diff
@@ -0,0 +1,32 @@
+import {
```

Comment:
I think the filename shouldn't have a capital letter, it feels like it's a component but it's an util file?

Reply:
You are right, thanks. I also renamed `Color` into `color` as it was a util file as well.